mkdocs.yml

```diff
@@ -246,7 +247,6 @@ plugins:
   - redirects:
       redirect_maps:
         'ftldns/database.md': database/index.md
-        'ftldns/regex/index.md': ftldns/regex/overview.md
```

Comment:
Why did you remove this line?

Reply:
It is a duplicate, and the target does not even exist. Further down, we have
```
'ftldns/regex/index.md': regex/index.md
```